### PR TITLE
Add KSM

### PR DESCRIPTION
**Pull request by JeremyParish69**
## What is the purpose of the change

To add the PICA and KSM tokens from Composable

## Brief Changelog

-adds PICA and KSM to chain-infos.ts, ibc-assets.ts, and defines prices for each, and adds token logos (both under 100 KB).

## Testing and Verifying

This change has been tested locally by rebuilding the website and verified content and links are expected

## Pending
KSM does not work. Will rebuild previews once Stage (which will have the 'Unsafe IBC Registrar') is merged to master brange.

### CODE REVIEW

File: packages/web/config/generate-chain-infos/source-chain-infos.ts [hunk 1]
```diff
@@ -3689,6 +3689,42 @@ const mainnetChainInfos: SimplifiedChainInfo[] = [
...
+    },
+    bech32Config: Bech32Address.defaultBech32Config("centauri"),
+    currencies: [
+      {
+        coinDenom: "PICA",
+        coinMinimalDenom: "ppica",
+        coinDecimals: 12,
+        coinGeckoId: "pool:ppica",
+        coinImageUrl: "/tokens/composable.png",
+        isStakeCurrency: true,
```
Comment (JeremyParish69) [inline packages/web/config/generate-chain-infos/source-chain-infos.ts:3708]: I'm getting the error 'stakeCurrency" is required, even though there is a stakeCurrency defined....

Comment (JoseRFelix) [reply packages/web/config/generate-chain-infos/source-chain-infos.ts:3708]: This should be fixed by using #1735 assets generation.

**Comment by JoseRFelix** [comment]
@JeremyParish69 Fix has been merged to master. This should be unblocked

File: packages/web/config/generate-chain-infos/source-chain-infos.ts (hunk 1 above)
Comment (JeremyParish69) [reply packages/web/config/generate-chain-infos/source-chain-infos.ts:3708]: Yes, it does--thanks!

**Review by jonator — APPROVED** [review]

**Comment by JeremyParish69** [comment]
No longer needed as KSM was added to stage

**Pull request closed without merging**